migration/src/app.ts

```diff
@@ -9,7 +9,7 @@ import MigrationService from "./migrate/service";
 export default class App {
   private logger = console;
   private pg = new Postgres("default");
-  private fb = new Firebase(config.get("firebase"));
+  private fb = new Firebase(firebaseConfig);
```

Comment:
u can just pass the 3 vars we require instead of the object

Reply:
hmm then i dont understand the decoupling, what are we trying to achieve here

Reply:
wait are we decoupling env from firebase

Reply:
the idea is u dont want the firebase class to be coupled to the env vars. if u pass the creds via the constructor, then u can have infinite amount of firebase connections, and the creds can come from anywhere. makes the code more flexible and reusable.

Reply:
fair enough fair enough, i was thinking purely in the context of our code, this makes the big senses now

Reply:
ah we gonna have to scuff it. in the postgres.ts add a typedef:
`type DBType = "postgres" | "mysql";`

and then 
`type: env.DB_TYPE as DBType`